src/background.js

```diff
@@ -31,8 +29,6 @@ const FACEBOOK_DOMAINS = [
   
   "bulletin.com", "facebookbrand.com",
 
-  "metacareers.com", "meta.com",  "metaque.st",
```

Comment:
See previous comment. Same issue here.

Reply:
This PR doesn't change the `background.js` file.